docs/rfcs/1020-pki-support.md

```diff
@@ -0,0 +1,291 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+# PKI Support
+
+## Overview
+
+Add support to use Parsec with an externally managed PKI
+
+## Background & Motivation
+
+Some organizations manage their own PKI, and want Parsec to rely on that PKI.
+
+That custom PKI come in the form of certificate with their private key often loaded in a dedicated device (a smartcard/token like a YubiKey).
+They would like that we use their certificate to access Parsec, so the user does not have to use a password or rely on the keyring of the OS.
+
+There is also an opportunity to rely on that PKI to join an organization.
+
+Put in a simpler term access Parsec using a Smartcard.
+
+### Terms:
+
+- **PKI**: Public Key Infrastructure
+- **Smartcard** or **Token**:
+  A tangible device that is similar to an USB stick (that store data).
+  That device is used to securely store private key, secrets and is able to perform cryptographic operations
+- **Submitter**: A user/device submitting a request to join an organization using the PKI feature.
+- **Accepter**: A user/device admin of an organization in charge of accepting or not the join request.
+
+## Goals and Non-Goals
+
+Some prior work was already done to add some primitives (sign, verify, encrypt, decrypt) to work with the windows certificate store under the `libparsec_platform_pki`.
+
+The goals are to define the higher elements to make the PKI work with Parsec for the invitation process, those higher elements are:
```

Comment:
So this is API is strictly based on what's been done in V2?

Or there is already some design/suggestions/ideas involved that make it somehow different from V2?